README.md

```diff
@@ -24,3 +24,47 @@
 
 - [x] ARYAN: I will prepare the dataset -- downloading + loading them up in notebook
 - [x] MKO: I will add some of the suggested topics (datasets) -- downloading + loading them up in notebook
+
+## Plan for pp
+
+How to be happier?
+Ask questions from some student's percpective.
+
+## Charcater sugestion
+Forrest Gump (from Forrest Gump):
```

Comment:
The only connection to `Forrest Gump` is his inability to "perceive or express happiness normally" because he is autistic or has a low IQ, right?
How well connected is this character to all the datasets we will explore?
I liked the Czech student character better--simple and without necessary "dependencies" to relate to references.
But, if you are confident about using Gump, I will go for it. :)

Reply:
I would not take Forrest Gump as a main character, I believe the usual student is better and more suitable